clang/tools/clang-linker-wrapper/ClangLinkerWrapper.cpp

```diff
@@ -1149,7 +1149,7 @@ wrapSYCLBinariesFromFile(std::vector<module_split::SplitModule> &SplitModules,
   return OutputFilePath;
 }
 
-/// Run llc tool for SYCL offloading.
+/// Run clang tool for SYCL offloading.
 /// 'InputFile' is the wrapped input file.
 /// 'Args' encompasses all arguments required for linking and wrapping device
 /// code and will be parsed to generate options required to be passed into the
```

Comment:
Please, update the next line as well.

Reply:
Right, thanks for spotting that, there were a few other comments in the same file as well, also updated.